jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -1006,3 +952,22 @@ def base_print(label, arr):
     print(label)
     print(arr_str)
 
+
+def dbg_print_data(model):
+    nints, ngroups, nrows, ncols = model.data.shape
+    print("Model Data:")
+    for row in range(nrows):
+        for col in range(ncols):
+            print(f"    Pixel ({row}, {col})")
+            for integ in range(nints):
+                print(f"[{integ}] {model.data[integ, :, row, col]}")
+
+
+def dbg_print_dq(model):
+    nints, ngroups, nrows, ncols = model.data.shape
+    print("Model DQ:")
+    for row in range(nrows):
+        for col in range(ncols):
+            print(f"    Pixel ({row}, {col})")
+            for integ in range(nints):
+                print(f"[{integ}] {model.groupdq[integ, :, row, col]}")
```

Comment:
not sure if these debug functions should remain permanently inside the repo, but I'm not too bothered either way personally. just something for maybe Tyler or Melanie to look at/decide

Reply:
I am fine with removing them.  For debugging and test development I keep around debugging statements that I find myself rewriting over and over, but I understand if people don't want these functions in the repo.